client/src/pages/CompactTreeView/V2CompactTreeGroupContainer.tsx

```diff
@@ -107,15 +107,28 @@ export class CompactTreeGroupContainerBase extends React.PureComponent<
       const didMaliciousDeclineToChallenge =
         isHonestOracleCurrentlyResolved && !normal;
 
+      const isAwaitingHonestDecision =
+        normal && normal.isAwaitingHonestExpertDecision;
+
+      const didHonestDecideToConcede =
+        normal && normal.isEligibleForHonestOracle && !isAwaitingHonestDecision;
+
+      const didHonestDecideToPlayOut =
+        normal &&
+        !normal.isEligibleForHonestOracle &&
+        !isAwaitingHonestDecision;
+
       const didHonestWin =
         didMaliciousDeclineToChallenge || // this is here b/c in some older trees the follow disjunct is false in cases where the malicious oracle declines
-        (isHonestOracleCurrentlyResolved &&
+        (didHonestDecideToPlayOut &&
+          isHonestOracleCurrentlyResolved &&
           isSamePointerInHonestAnswerDraftAndMaliciousQuestion);
 
       const didMaliciousWin =
-        !didHonestWin && // this is here b/c in some older trees the follow two conjuncts are true in cases where the malicious oracle declines
-        isHonestOracleCurrentlyResolved &&
-        !isSamePointerInHonestAnswerDraftAndMaliciousQuestion;
+        (!didHonestWin && // this is here b/c in some older trees the follow two conjuncts are true in cases where the malicious oracle declines
+          isHonestOracleCurrentlyResolved &&
+          !isSamePointerInHonestAnswerDraftAndMaliciousQuestion) ||
+        didHonestDecideToConcede;
```

Comment:
The win logic is complex, but it seems fair as the underlying system is complex. I'd like to detect a situation where both `didHonestWin` and `didMaliciousWin` are `true`. Could we add a check here?

Reply:
Sure